src/ExpressServer.ts

```diff
@@ -0,0 +1,44 @@
+import express from "express";
+import { Environment } from "./utils/constants";
+import { SetHueTokens } from "./utils/helpers";
+
+export default class ExpressServer {
+    private _server;
+
+    constructor() {
+        this._server = express();
+
+        this._server.get("/", (request, response) => {
+            response.send("Discord bot active.");
+        });
```

Comment:
This message doesn't seem particularly useful. What is our objective here?

Reply:
@TheRageKage12 this is actually required by azure web apps - it needs to respond with a 200 (something) to the root URL at port 8080 or something in order to check container health.

Reply:
In the future we can add a full-on Express router for frontend, so this can also be considered a placeholder for the upcoming web frontend.